lv_draw/lv_draw_rbasic.c

```diff
@@ -305,18 +305,13 @@ void lv_rmap(const lv_area_t * cords_p, const lv_area_t * mask_p,
         for(row = masked_a.y1; row <= masked_a.y2; row++) {
             for(col = masked_a.x1; col <= masked_a.x2; col++) {
                 lv_color_t * px_color = (lv_color_t *) &map_p[(uint32_t)(col - masked_a.x1) * sizeof(lv_color_t)];
-
                 if(chroma_key && chroma_key_color.full == px_color->full) continue;
-
                 if(recolor_opa != LV_OPA_TRANSP) {
                     lv_color_t recolored_px = lv_color_mix(recolor, *px_color, recolor_opa);
-
                     lv_rpx(col, row, mask_p, recolored_px, LV_OPA_COVER);
-                } else {
-                    lv_rpx(col, row, mask_p, *px_color, LV_OPA_COVER);
```

Comment:
Why does a 'recolored' pixel still get drawn the old way, but a non-recolored pixel doesnt?

Reply:
The reason I ask, is I believe some of our graphic elements go through "recoloring" and some don't. I believe the recolored pixels need to be buffered and drawn via the same "image draw" as the non-recolored pixels. @mechanikalk  can confirm.